.github/workflows/test-go.yml

```
@@ -239,17 +239,24 @@ jobs:
           fi
 
           # shellcheck disable=SC2086 # can't quote package list
+
+          # On a release branch, add a flag to rerun failed tests
+          if [[ ${{ github.base_ref }} = release/* ]] ; then
+            rerunFails="--rerun-fails"
```

Minor nit: to stay consistent with variable names, maybe we should name this `RERUN_FAILS` instead?
```suggestion
           RERUN_FAILS="--rerun-fails"
```